docs/config_entries_config_flow_handler.md

```diff
@@ -59,8 +59,10 @@ There are a few step names reserved for system use:
 | `mqtt`      | Invoked if your integration has been discovered via MQTT as specified [using `mqtt` in the manifest](creating_integration_manifest.md#mqtt).             |
 | `ssdp`      | Invoked if your integration has been discovered via SSDP/uPnP as specified [using `ssdp` in the manifest](creating_integration_manifest.md#ssdp).             |
 | `usb`       | Invoked if your integration has been discovered via USB as specified [using `usb` in the manifest](creating_integration_manifest.md#usb).             |
-| `user`      | Invoked when a user initiates a flow via the user interface or when discovered and the matching and discovery step are not defined.                                                                                                  |
+| `user`      | Invoked when a user initiates a flow via the user interface or when discovered and the matching and discovery step are not defined.                           |
 | `zeroconf`  | Invoked if your integration has been discovered via Zeroconf/mDNS as specified [using `zeroconf` in the manifest](creating_integration_manifest.md#zeroconf). |
+| `reauth`    | Invoked if your integration indicates it [requires reauthentication, e.g., due to expired credentials](integration_setup_failures#handling-expired-credentials).  |
+| `import`    | Invoked if [YAML configuration is found](configuration_yaml_index), this can be used to migrate old YAML configuration. |
```

Comment:
I worry this makes it read like the system is going to invoke this step if it finds yaml configuration, which isn't the case. (Like re-auth, the integration needs to find the yaml and decide what to import).  The link to that page does generally explain yaml, but does not explain really how import works.  Given import is rare (only used for older existing integrations, and not used for new integrations) i'm not sure its totally worth documenting unless we're also actually explaining the import steps to get an old integration into shape.

Reply:
Ohh, I didn't realize that it isn't automatically called, I'm not yet there on my journey... :-) I just wanted to mention it there somehow so that one doesn't re-use the name for a regular step, potentially causing some unexpected, hard-to-debug behaviors.

Reply:
I simplified this to only note that this is reserved for migrations, what do you think?